src/modules/Auth/containers/onboarding/Causes/useCauses.tsx

```diff
@@ -1,18 +1,20 @@
 import { useContext, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
 import { SOCIAL_CAUSES } from 'src/constants/SOCIAL_CAUSES';
+import { translate } from 'src/core/utils';
 import { StepsContext } from 'src/modules/Auth/containers/onboarding/Stepper';
 import { useUser } from 'src/modules/Auth/contexts/onboarding/sign-up-user-onboarding.context';
 type SocialCauseVal = {
   label: string;
   value: string;
 };
 export const useCauses = () => {
-  const keytems = Object.keys(SOCIAL_CAUSES);
+  const keyItems = Object.keys(SOCIAL_CAUSES);
   const [value, setValue] = useState<SocialCauseVal[]>([]);
   const { state, updateUser } = useUser();
   const { updateSelectedStep } = useContext(StepsContext);
-  const items = keytems.map(i => {
-    return { value: SOCIAL_CAUSES[i].value, label: SOCIAL_CAUSES[i].label };
+  const items = keyItems.map(i => {
+    return { value: SOCIAL_CAUSES[i].value, label: translate(SOCIAL_CAUSES[i].value) };
```

Comment:
i didn't check it, should it be `.value` for label?

Reply:
Yes, keys are used for translation